rewrite-java/src/main/java/org/openrewrite/java/ChangePackage.java

```diff
@@ -27,6 +27,9 @@
 
 import java.nio.file.Paths;
 
+/**
+ * Deprecated, refer to RenamePackage.
```

Comment:
Suggest we elaborate on the original functionality of `ChangePackage` to include this behavior rather than renaming it unnecessarily? Are these two recipes different?

As an aside, if we ever do deprecate something, it's preferable to use the `@Deprecated` annotation that is recognized by IDEs rather than a comment.

Reply:
Move the code into ChangePackage, and remove the changes related to RenamePackage.